docs/DeveloperGuide.md

```diff
@@ -92,20 +92,43 @@ The `UI` component:
+<div markdown="block" class="alert alert-info"> 
+:information_source: **Note:** ABC refers to `Deliverable`, `Meeting`, `Person`. 
+XYZ refers to any command call, e.g Add, find, delete, etc. 
+
+</div>
+
+**API** : <br>
+* [`LogicDeliverable.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicDeliverable.java)
+* [`LogicMeeting.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicMeeting.java)
+* [`LogicPerson.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicPerson.java)
+* [`LogicDispatcher.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicDispatcher.java)
+
+The `Logic` component parses the user commands and executes them. There are different pa`LogicDispatcher` selects the correct parser to 
+
+This is the list of what `Model` components affected by their respective `Logic` components: 
+* `LogicDeliverable`: Component that affects `DeliverableModel`.
+* `LogicMeeting`: Component that affects `MeetingModel`.
+* `LogicPerson`: Component that affects `PersonModel`.
 
-1. `Logic` uses the `AddressBookParser` class to parse the user command.
-1. This results in a `Command` object which is executed by the `LogicManager`.
+The components follow the general sequence to execute a command:
+
+1. `Logic` uses the `XYZBookParser` class to parse the user command.
+1. This results in a `XYZCommand` object which is executed by the `LogicXYZManager`.
 1. The command execution can affect the `Model` (e.g. adding a person).
 1. The result of the command execution is encapsulated as a `CommandResult` object which is passed back to the `Ui`.
 1. In addition, the `CommandResult` object can also instruct the `Ui` to perform certain actions, such as displaying help to the user.
 
-Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("delete 1")` API call.
+Given below is the Sequence Diagram for interactions within the `Logic` component for API call of any command. 
 
-![Interactions Inside the Logic Component for the `delete 1` Command](images/DeleteSequenceDiagram.png)
+![Interactions Inside the Logic Component for any command](images/CommandSequenceDiagram.png)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+<div markdown="span" class="alert alert-info">:information_source: 
+**Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) 
+but due to a limitation of PlantUML, the lifeline reaches the end of diagram.<br>
+</div>
+<div markdown="span" class="alert alert-primary">:bulb:
+**Tip**: Refer to [Switch Mode Feature](#switch-mode-feature) below for more details on the Switch Mode feature.
```

Comment:
Im not sure but I think the Logic component section doesn't mention switch command? So this tip might be coming out of nowhere

Reply:
Initially I had mentioned it in the description but removed it. Wil change accordingly